doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -217,8 +217,16 @@ Restart CORTX
 2. Login to the CORTX OVA as cortx and run the following.
 
    - **sudo su -**
+   
+3. Restart openldap and s3 auth server services by the below mentioned commands.
```

Comment:
Why are these hard-coded?  They should be #. and let RST do the dynamic numbering.

Reply:
It will be corrected.

Reply:
Updated and Resolved.